pkg/test/fake_credential_secret_store.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	corev1api "k8s.io/api/core/v1"
+)
+
+// SecretStore defines operations for interacting with credentials
+// that are stored in Secret.
+type SecretStore interface {
```

Comment:
Why define the same interface again here?

And it's better to generate the mock implementation by the [mockery tool](https://github.com/vektra/mockery)

Reply:
OK, will use mockery.
I added in this way just followed the existing file_store.go. But for new code, yes, let's use the better way.

Reply:
Done